content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -248,17 +248,18 @@ However, the particular path specified in the custom recycler Pod template in th
 `volumes` part is replaced with the particular path of the volume that is being recycled.
 
 ### PersistentVolume deletion protection finalizer
-{{< feature-state for_k8s_version="v1.23" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.31" state="beta" >}}
```

Comment:
Consider using the "feature_gate_name" option for feature-state shortcode. This will make it more data-driven by pulling from the feature gate description file.

```suggestion
{{< feature-state feature_gate_name="HonorPVReclaimPolicy" >}}
```

Reply:
Removed this from the blog MR